### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by davidkrivko**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,49 @@
...
+        self.create_powerful_api()
+        self.create_awesome_web_page()
+
+
+full_stack_dev = FullStackDeveloper("Tom")
+print(full_stack_dev.skills)
+full_stack_dev.create_web_application()
+
+android_dev = AndroidDeveloper("Beth")
+print(android_dev.skills)
```
Comment (DmytroSvirsa) [inline app/main.py:49]: remove unused code

File: app/main.py [hunk 2]
```diff
@@ -1 +1,49 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```
Comment (DmytroSvirsa) [inline app/main.py:2]: make it as self attribute

**Review by DmytroSvirsa — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 2 above)
Comment (davidkrivko) [reply app/main.py:2]: put it in the constructor?

Comment (Abnormaltype) [reply app/main.py:2]: yep

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
put in the init

**Pull request closed without merging**